recipes/coin3d/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "coin3d" %}
+{% set version = "4.0.0" %}
+{% set sha256 = "8319cfa699afac0e5f19054222c25a7ad655f018f7c3c7b53f626e7b167cb08e" %}
+{% set download_url = "https://github.com/FreeCAD/pivy/releases/download/0.6.3/Coin3D-coin-8e8337f61b05.tar.gz" %}
```

Comment:
Can you use official releases?

Reply:
I will change to official release once there is one available. This is a snapshot from the bit-bucket repo.

Reply:
Can you change this to `https://bitbucket.org/Coin3D/coin/get/{{ commit }}.tar.gz` where `commit` is set to `8e8337f61b05`?

Reply:
this should be possible for the coin-recipe, but I encountered problems for repositories with sub-modules like the other recipe I made a PR for (simage).